File: Java/Data Structures & Algorithms Module - Java/Revision/Stack/ArrayStackApp.java
/*
ArrayStackApp.java
Billy Cussen
07/01/2021
*/
public class ArrayStackApp {
  
    public static void main(String [] args){
        ArrayStack<Integer> myStack = new ArrayStack<Integer>(2);
        myStack.push(2);
        myStack.push(4);
        myStack.push(6);
        myStack.push(8);
        myStack.push(10);

        System.out.println(myStack.toString());

        myStack.pop();
        myStack.pop();
        System.out.println("Size: "+myStack.size());
        System.out.println("Peek: "+myStack.peek());
        System.out.println(myStack.toString());        
    }

}

File: Python/Data Structures & Algorithms Module - Python/Revision/Stacks/ArrayStackApp.py
"""
ArrayStackApp.py
Billy Cussen
07/01/2021
"""
from ArrayStack import ArrayStack

myStack = ArrayStack()

print("Is Empty: "+str(myStack.isEmpty()))
print("Size: "+str(myStack.getSize()))
try:
    print("Peek: "+str(myStack.peek()))
except Exception as e:
    print(e)
try:
    print("Pop: "+str(myStack.pop()))
except Exception as e:
    print(e)
    
try:
    print(myStack.toString())
except Exception as e:
    print(e)

myStack.push(2)
myStack.push(4)
myStack.push(6)
myStack.push(8)

print("Is Empty: "+str(myStack.isEmpty()))
print("Size: "+str(myStack.getSize()))
try:
    print("Peek: "+str(myStack.peek()))
except Exception as e:
    print(e)
try:
    print("Pop: "+str(myStack.pop()))
except Exception as e:
    print(e)
    
try:
    print(myStack.toString())
except Exception as e:
    print(e)

